app/connectors/AddressLookupFrontendConnector.scala

```diff
@@ -22,7 +22,7 @@ import models.addresslookup.{AddressLookupConfirmation, AddressLookupOnRamp, Add
 import play.api.http.HeaderNames.LOCATION
 import play.api.http.Status
 import uk.gov.hmrc.http.HttpReads.Implicits._
-import uk.gov.hmrc.play.bootstrap.http.HttpClient
+import uk.gov.hmrc.http.HttpClient
```

Comment:
wasnt expecting this class (or the NDRCConnector class) to change

Reply:
This was fixing a deprecated import for HTTPClient (caused by the recent Play2.8 upgrade)